packages/wallets/src/components/Base/WalletTextField/__tests__/WalletTextField.spec.tsx

```diff
@@ -0,0 +1,111 @@
+import React, { createRef } from 'react';
+import { fireEvent, render, screen } from '@testing-library/react';
+import HelperMessage from '../HelperMessage';
+import WalletTextField, { WalletTextFieldProps } from '../WalletTextField';
+
+jest.mock('../HelperMessage', () => jest.fn(() => null));
+
+describe('WalletTextField', () => {
+    const defaultProps: WalletTextFieldProps = {
+        defaultValue: '',
```

Comment:
minor - possibly not tested property, defaultValue

Reply:
Updated